### ISSUE TITLE
Wrong location of death is shown

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#315]
GVExport shows in the latest development version instead of the place of death the place where the person is buried.

Many of the places where persons in my tree are buried are cemeteries. Now it looks like those people died at a graveyard.

**Comment by Neriderc** [comment]
Is this new?

From my testing, if there is a place of death it is used. If there is no place of death but there is a place of burial, that is used.

This seems to be a webtrees thing, the place of death in GVE always comes from $individual->getDeathPlace()->gedcomName() and this has not been changed recently.

From what I can tell in the webtrees code, deaths, burials, and cremations are counted as death events. It seems to just grab the first one:
![image](https://user-images.githubusercontent.com/89974636/210458764-9a4ab7d9-4174-42c6-bf05-64ff4f348988.png)

The order seems to come from general webtrees ordering code (GedcomRecord->facts() holds this). For the individuals that you see the burial place, do they also have a place of death? Does this show in the right order in webtrees?

My understanding (which may well be wrong) is that the place of death is a town, and the address was the more specific address such as the graveyard. 

So for example, William Shakespeare is buried at The Collegiate Church of the Holy and Undivided Trinity, Stratford-upon-Avon, Warwickshire, England. So I would enter the place of burial as "Stratford-upon-Avon, Warwickshire, England" and the address as "The Collegiate Church of the Holy and Undivided Trinity, Stratford-upon-Avon, Warwickshire, England". This is based on my understanding of the GEDCOM standard. When entered like this, it makes sense to use the place of burial where the place of death is not known, and it does not list any graveyards as this detail is in the address field not the place of burial field.

**Comment by hartenthaler** [comment]
I assume now that this is not new. I observed it the first time because I analyzed a part of my tree with several persons with unknown place of death, but known place of burial. 

Your analysis is perfect - as always - thank you!

Some time ago I startet to use shared locations (_LOC) using the Vesta modules. Then I defined at the lowest hierarchy level of locations not only towns and villages but also farm houses, graveyards and so on.  And in order to keep PLAC synchrone to the shared locations I have now for example "PLAC The Collegiate Church, Stratford-upon-Avon, Warwickshire, England, GBR". I will solve this issue by adding a place of death; for example "Germany" if I know that somebody died in Germany but don't know the place.
